source/src/main/java/org/cerberus/database/DatabaseVersioningService.java

```diff
@@ -8799,6 +8799,12 @@ public boolean isDatabaseUptodate() {
         SQLS.append("INSERT INTO `invariant` (`idname`, `value`, `sort`, `description`, `VeryShortDesc`) VALUES ('USERGROUP', 'TestStepLibrary', '115', 'Can modify Step Library and flag Step as Library.', '');");
         SQLInstruction.add(SQLS.toString());
 
+        // New design Login page
+        //-- ------------------------ 1158
+        SQLS = new StringBuilder();
+        SQLS.append("UPDATE `parameter` SET value='<a href=\"mailto:support@domain.com?Subject=Cerberus%20Account\">Support</a>' where param='cerberus_support_email'");
```

Comment:
I better understand and don't have any better solution ;-(.
Maybe try to get current email value by regex inside the SQL ?
I would also add just like you proposed the mention to check that parameter value is correctly after deploy that version of Cerberus.